app.py

```diff
@@ -48,7 +48,26 @@ def experience():
         return jsonify()
 
     if request.method == 'POST':
-        return jsonify({})
+
+        request_body = request.get_json()
+
+        new_experience = Experience(
+            request_body['title'],
+            request_body['company'],
+            request_body['start_date'],
+            request_body['end_date'],
+            request_body['description'],
+            request_body['logo']
```

Comment:
nit: What if any of these fields are missing? How would the code handle such unexpected situation? What if a relatively important field like title or company is missing or invalid?

This is a nit, as it is not necessary as per the ticket you're aiming to resolve. But it is a good place to have this discussion, we can go with the code like should you think so, but it's a nice thing to keep in mind.

Reply:
I'd like to work on fixing this nit before closing the PR if that's okay. I want to improve my coding skills so I will try to add some error handling here in the event of a bad request.

Reply:
If I understand correctly, I should add error handling for four possible areas of failure.

1. The request isn't JSON
2. The request is missing required fields (Fields that should be there but are not)
3. The fields entered are the wrong data type (Fields that are present but are not the right type)
4. The JSON contains fields that do not match the model (Fields that are there but should not be)

Is this a good understanding of the areas of failure?

Reply:
I have implemented error handling now for the first three areas of concern. Let me know if you would like any other changes to be made before closing the PR.

Reply:
One more thing, not part of this PR but commenting it here since it's an interesting thing to keep in mind. Sometimes giving away too much information in the errors is bad for software. This can be abused by potential attackers, who can distinguish and use some logical reasonings to determine your internal code and find a way to break it.

Again, not something that needs to be done here, just an interesting fact.